CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 ### Fixed
-- Some web fonts were blocked which caused icons to be missing
+- Unable to back out of YouTube with remote back button
```

Comment:
"An issue where users were unable to leave YouTube with remote back button (#...)"

Include a bug number and perhaps elaborate more, like ^